Dockerfile

```diff
@@ -307,8 +318,11 @@ RUN \
   libdavs2: env.DAVS2_VERSION, \
   libxavs2: env.XAVS2_VERSION, \
   libmodplug: env.LIBMODPLUG_VERSION, \
-  uavs3d: env.UAVS3D_COMMIT, \
+  libuavs3d: env.UAVS3D_COMMIT, \
```

Comment:
I've used to same naming as ffmpeg here but maybe better to name things similar to what the project uses?

Reply:
i do agree!

Reply:
Use ffmpeg naming or project name? :) think ill merge and we can fix thing later

Reply:
right, unclear. i meant either way, as long as there is a recognizable standard that makes sense